src/Polly.Core/ResiliencePipelineBuilderBase.cs

```diff
@@ -54,6 +54,18 @@ private protected ResiliencePipelineBuilderBase(ResiliencePipelineBuilderBase ot
     /// </value>
     public string? InstanceName { get; set; }
 
+    /// <summary>
+    /// Gets or sets the context pool associated with the builder.
+    /// </summary>
+    /// <remarks>
+    /// A custom pool can be used to configure custom behavior for ResilientContext creation.
```

Comment:
Can you make the type a reference rather than just text, then it should link through in our API docs.

Reply:
will do!  that was a miss

Reply:
done